blog/robots-101.html

```diff
@@ -0,0 +1,20 @@
+---
+layout: blog
+title: Robots 101
+---
+
+<p>Welcome to Robot 101, our series where we guide you through the competition and give some advice on your next steps.</p>
+
+{% for page in site.tags.robots_101 %}
+  {% include get_description_for_page.html %}
```

Comment:
I'm not sure that using the _description_ here is quite right? This is intended to be part of the metadata for a page, exposed in header information. It falls back to the site description for example.

I suggest instead that we use `page.excerpt` directly, processing it as needed.

I also wonder if it would be worth us introducing a span or something around the "welcome to the series" blocks on each page which this can then skip over to avoid repetition and give more of a feel for the actual page content.